device-registry/src/main/scala/org/genivi/sota/device_registry/GroupsResource.scala

```diff
@@ -88,6 +111,12 @@ class GroupsResource(namespaceExtractor: Directive1[Namespace])
         (get & path("devices")) {
           getDevicesInGroup(groupId)
         } ~
+        (post & pathPrefix("devices") & namespaceExtractor & extractUuid) { (ns, deviceId) =>
```

Comment:
Instead of extractUuid, make a new extractDeviceUuid extractor with the allowExtractor.